configure.ac

```diff
@@ -0,0 +1,191 @@
+        if test x"${GPG}" == x ; then
+            AC_MSG_ERROR([gpg not found])
+        fi
+
+        AC_CHECK_PROG([MKTEMP],[mktemp],[mktemp])
+        if test x"${MKTEMP}" == x ; then
+            AC_MSG_ERROR([mktemp not found])
+        fi
+
+        AC_CHECK_PROG([MD5SUM],[md5sum],[md5sum])
+        if test x"${MD5SUM}" == x ; then
+            AC_MSG_ERROR([md5sum not found])
+        fi
+
+        AC_CHECK_PROG([CPIO],[cpio],[cpio])
+        if test x"${CPIO}" == x ; then
+            AC_MSG_ERROR([cpio not found])
+        fi
+
+        AC_CHECK_PROG([GZIP],[gzip],[gzip])
+        if test x"${GZIP}" == x ; then
+            AC_MSG_ERROR([gzip not found])
+        fi
+
+        AC_CHECK_PROG([UNSQUASHFS],[unsquashfs],[unsquashfs])
+        if test x"${UNSQUASHFS}" == x ; then
+            AC_MSG_ERROR([unsquashfs not found])
+        fi
+
+        AC_CHECK_PROG([SORT],[sort],[sort])
+        if test x"${SORT}" == x ; then
+            AC_MSG_ERROR([sort not found])
+        fi
+        
+		;;
+	src)
+        AC_MSG_RESULT([stage1 will build systemd from source])
+
+        # these are needed for systemd build
+        AC_CHECK_PROG([INTLTOOLIZE],[intltoolize],[intltoolize])
+        if test x"${INTLTOOLIZE}" == x ; then
+            AC_MSG_ERROR([intltoolize not installed, apt-get install intltool])
+        fi
+
+        AC_CHECK_PROG([LIBTOOLIZE],[libtoolize],[libtoolize])
+        if test x"${LIBTOOLIZE}" == x ; then
+            AC_MSG_ERROR([libtoolize not installed, apt-get install libtool])
+        fi
+		;;
+	*)
+		AC_MSG_ERROR([stage1 build type '${RKT_STAGE1_USR_FROM}' unsupported])
+esac
+
+AC_SUBST(RKT_STAGE1_USR_FROM)
+
+## STAGE1: Systemd git path
+AC_ARG_WITH(stage1-systemd-src,
+        AS_HELP_STRING([--with-stage1-systemd-src=git-path],
+                [address to git repository of systemd, used in 'src' build mode]),
+        [RKT_STAGE1_SYSTEMD_SRC="$withval"],
+        [RKT_STAGE1_SYSTEMD_SRC="git://anongit.freedesktop.org/systemd/systemd"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_SRC)
+
+## STAGE1: Systemd version
+AC_ARG_WITH(stage1-systemd-version,
+        AS_HELP_STRING([--with-stage1-systemd-version=version],
+                [systemd version to build]),
+        [RKT_STAGE1_SYSTEMD_VER="$withval"],
+        [RKT_STAGE1_SYSTEMD_VER="v219"])
+
+AC_SUBST(RKT_STAGE1_SYSTEMD_VER)
+
+## STAGE1: linker-defined custom STAGE1 image path, default is unset
+AC_ARG_WITH(stage1-image-path,
+        AS_HELP_STRING([--with-stage1-image-path],
+                [custom stage1 image path]),
+        [RKT_STAGE1_IMAGE="$withval"],
+        [RKT_STAGE1_IMAGE=""])
+
+RKT_STAGE1_IMAGE_FLAGS=
+# if stage1 image variable is set, add a linker flag to rkt defining the variable
+if test x"${RKT_STAGE1_IMAGE}" != x ; then
+   RKT_STAGE1_IMAGE_FLAGS="-ldflags \"-X main.defaultStage1Image '${RKT_STAGE1_IMAGE}'\""
+fi
+
+AC_SUBST(RKT_STAGE1_IMAGE_FLAGS)
+
+## STAGE1 D-Bus parameters
+AC_ARG_WITH(stage1-dbus,
```

Comment:
Actually, reverted it back, because it is looks clearer with AC_ARG_WITH because it ends up in "Optional PACKAGES" section, instead of "Optional Features" section.